lib/index.js

```diff
@@ -53,6 +53,7 @@ function defaults(options) {
   options || (options = {});
   options.apiKey || (options.apiKey = 'YOUR_API_KEY');
   options.host || (options.host = 'cdn.segment.com');
+  options.ajsPath || (options.ajsPath = '/analytics.js/v1/\' + key + \'/analytics.min.js');
```

Comment:
@dk1027 is `key` a known variable?

Reply:
@pooyaj `key` is a variable in the scope of where `settings.ajsPath` is renderered: https://github.com/segmentio/snippet/pull/50/files#diff-7ca0ae40f9ef9562ed8a5fcdce4f82d2R71